src/utils/user.ts

```diff
@@ -0,0 +1,48 @@
+import cheerio, { CheerioAPI } from "cheerio";
+
+import axios from "axios";
+
+interface UserProfile extends CheerioAPI {
+  kind: "UserProfile";
+}
+
+export interface User {
+ id: string;
+ name: string;
+ pseuds: string;
+ url: string;
+ joined: string;
+ bioHtml?: string | null;
```

Comment:
very last thing, maybe let's just type this as `bioHtml: string | null`.

This basically says: this key will always be present, but it might be null. `bioHtml?: string | null` also covers the case where `bioHtml` isn't there at all.

Reply:
That answers my question from earlier, will do